### PR TITLE
Package items

### LINKED ISSUE #114: Cannot browse dataset list if there are many datasets
The datasets section of the portal does not show any kind of pagination controls. This means it is not possible to see all existing datasets when there is a large number of them

### LINKED ISSUE #127: There is no way to access dataset details
The current version of the portal UI does not allow one to access the dataset details page after performing a search. There is no clickable link anywhere on a list item to navigate to the detail page, as seen on the below screenshot:

image

### PR DESCRIPTION
**Pull request by Fanevanjanahary**
fixes #114 
fixes #127

- [x] Add pagination
- [x] Add the link detail on item title

### CODE REVIEW

**Review by ricardogsilva — CHANGES_REQUESTED** [review]
Seems to work OK. However, the link to the item detail page is perhaps too subtle. Please make the hyperlink more pronounced.

My suggestion is to make hyperlinks use the green color that is used elsewhere to denote buttons and links, such as in the search filters:

image

Don't make them bold though

**Comment by Fanevanjanahary** [comment]
hi @ricardogsilva please is that ok or does the font-weight need to be lower?

image

File: ckanext/dalrrd_emc_dcpr/logic/action/dcpr.py [hunk 1]
```diff
@@ -75,6 +75,7 @@ def dcpr_request_create(context, data_dict):
 
     try:
         model.Session.add(request)
+        model.repo.commit()
```
Comment (ricardogsilva) [inline ckanext/dalrrd_emc_dcpr/logic/action/dcpr.py:78]: This change seems totally unrelated to this PR. I guess you took it from one of the pending PRs. Please revert this, as I don't want to mix backend and frontend changes in this issue

**Review by ricardogsilva — CHANGES_REQUESTED** [review]
Almost good, I'd just ask that you revert the backend change, which is unrelated to this issue

**Review by ricardogsilva — APPROVED** [review]
Nice, thanks!

**Pull request merged**